examples/internal/proto/examplepb/a_bit_of_everything.proto

```diff
@@ -641,8 +641,7 @@ service ABitOfEverythingService {
 	}
 	rpc PostWithEmptyBody(Body) returns (google.protobuf.Empty) {
 		option (google.api.http) = {
-			post: "/v2/example/postwithemptybody/{name}",
-			body: "*"
```

Comment:
Lets revert this change too, apparently this is behaviour we want to keep working, judging by https://github.com/grpc-ecosystem/grpc-gateway/pull/527. Sorry for the confusion.

Reply:
No problem, I'll change it